doc/spec/manifest.md

```diff
@@ -0,0 +1,244 @@
+    to the application. Values should be a [SPDX License Identifier](https://spdx.org/licenses/).
+
+- **`Dependencies`** *array*
+
+    An array of objects which describe content which this application depends
+    on. Those may be generic blobs of data or other content manifests and
+    their dependencies as described by their media type. Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the dependency object.
+    
+    - **`size`** *int*
+    
+        The size in bytes of the dependency object.
+    
+    - **`repository`** *string*
+    
+        The name of the repository from which to fetch the object.
+
+    - **`encryption`** *object*
+
+        An object containing information about the algorithm, key, and
+        initialization vector used to decrypt the content. If omitted,
+        the content should be assumed to be unencrypted. Fields are:
+
+        - **`algorithm`** *string*
+
+            Identifies the  encryption algorithm used.
+
+        - **`key`** *string*
+
+            The hex encoding of the key required to decrypt the content.
+
+        - **`iv`** *string*
+
+            The hex encoding of the initialization vector required to decrypt
+            the content.
+    
+    - **`digest`** *string*
+    
+        A string labeled with a digest algorithm prefix and the hex-encoded
+        value of that digest for the content. If the content is encrypted,
+        then the digest is of the encrypted content.
+
+        Digest strings should be of the form:
+
+            <digest-alg-name>:<hex-encoded-digest>
+    
+    Any media type may be in the `mediaType` field as the application may
+    use any type of content. If the `mediaType` is
+    `application/vnd.docker.distribution.content.manifest.v2+json` then the
+    dependency is another content manifest as described by this document. The
+    dependencies from that manifest should also be considered dependencies of
+    this application.
+    
+    If the `repository` field is empty or omitted, then the object should be
+    considered to be in the same repository as this manifest.
+
+    Encryption combined with a verifiable digest of the content should allow
+    for both confidentiality and authentication of data.
+
+    In general, the ordering of these dependencies is significant as it may be
+    significant to the application.
+
+- **`application`** *object*
+
+    Fields are:
+    
+    - **`mediaType`** *string*
+    
+        The MIME type of the JSON object in the `data` field.
+    
+    - **`data`** *object*
+    
+        Abstract application data in any schema, as described by the
+        `mediaType` field.
+    
+    Application data may be *any* JSON object, allowing this manifest format,
+    along with the generic dependencies list, to support any type of 
+    application that can be represented as a JSON configuration and a
+    collection of content-addressable blobs of data.
+
+- **`metadata`** *object*
```

Comment:
why is there no shed emoji? I'll change it.

![1098-633918926907187500_wgb_cm-scale_w-800_h-710_q-80_wgb](https://cloud.githubusercontent.com/assets/1441929/6401081/74a52290-bdb2-11e4-8b2d-e5c68b782288.jpg)